templates/cisco_ios_show_ip_cef.textfsm

```diff
@@ -0,0 +1,43 @@
+#
+# This template is only compatible with the following command
+#   - show ip cef [vrf NAME]
+# This is enforced by INDEX file
+# Check cisco_ios_show_ip_cef_detail.textfsm for more details
+#
+Value Required PREFIX ((?:\d{1,3}\.){3}\d{1,3}\/\d{1,2})
+Value TYPE ((?:no\s)?(?:[a-z]+))
```

Comment:
We typically use the headers for the values as well. I don't think we should be capturing rows from the same column as different values (TYPE and NEXTHOP). They both are under the nexthop column.

Reply:
I am don't think I understand this suggestion. I've read few times, but still blind.

Reply:
The headers of the output are:
`Prefix               Next Hop             Interface`

I do not see type in there at all. You're saying there is either a "type" or "next hop" within the "Next Hop" column, but that is your assumption/use-case and not what the output is saying. You're deriving the type of not being an IP address and the next hop is an IP address.

This assumption/use-case should be handled in post-processing as that is an opinionated view of the data.

Your assumptions/use-cases are not wrong, but as stated above, it is opinionated and our repo attempts to stay as neutral as possible when it comes to parsing the data.

We should just stick to the headers and leave any use-cases to post processing once the data is structured.

Reply:
Ah, ok, I am with you now... so you're suggesting that there's no such thing as TYPE, but only NEXTHOP even though it is 'drop' or 'glean' or whatever. But in this case output from 'show ip cef' will not be compatible with 'show ip cef details' but if you want it this way I can do it

Reply:
This is resolved. Check last commit.